src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -398,7 +398,7 @@
 {# Support #}
 
 {%- block form_rows -%}
-    {% for child in form %}
+    {% for child in form if not child.rendered %}
```

Comment:
Should we really do this change? Better get the deprecation and then the exception if you're trying to render the same row twice?

Reply:
isn't this required to support partial form auto-rendering?

Reply:
It's already supposed to be handled in `form_rest`:
https://github.com/symfony/symfony/blob/4a7e6fa9bcc3f3aebffaa60a357b65054f8d6fa4/src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig#L376-L381

Reply:
This change is necessary for tests to pass

Reply:
Which test? If the test is failing by triggering the deprecation, that's expected and should by marked as legacy, right?

Reply:
It was triggered by FormExtensionDivLayoutTest, inherited testcase testRestWithChildrenForms.

Negative of just marking tests as legacy is that it triggers unresolvable deprecation notices in user land. There should be some effort in avoiding it, not just mark test as legacy as first thing IMHO.

I might have found better way though